lib/Textmaster/Model/Project.php

```diff
@@ -319,6 +320,36 @@ public function addDocuments(array $documents)
     /**
      * {@inheritdoc}
      */
+    public function getPotentialAuthors($status = null)
+    {
+        if (null === $this->getId()) {
+            throw new BadMethodCallException(
+                'The project must be saved before getting authors who can do it.'
+            );
+        }
+
+        $response = $this->getApi()->authors($this->getId())->all($status);
+
+        $nextPage = $response['next_page'];
```

Comment:
@cdaguerre 

>Is next_page always set or should we check for existence?

Yes, the value is either `null` or an `integer`  

>This code can be rewritten to avoid code duplication

Done

>Also, this could be abstracted to get iterable results for similar api routes

I disagree, if the call was `$this->getApi()->all($filters);` why not be here we do `$this->getApi()->authors($id)->all($status);` so I don't think there will be any similar route.